src/aws/flb_aws_util.c

```diff
@@ -581,6 +581,8 @@ void flb_aws_print_error(char *response, size_t response_len,
 
     error = flb_json_get_val(response, response_len, "__type");
     if (!error) {
+        /* error can not be parsed, print raw response */
+        flb_plg_error(ins, "Raw response: %s", response);
```

Comment:
This might not be the right way to think about things, but one interesting thought I just had- we already have folks complaining about the rates of hits for "error" from Fluent Bit: https://github.com/aws/aws-for-fluent-bit/blob/mainline/troubleshooting/debugging.md#rate-of-network-errors

So I wonder if this should be warn, because its printing a raw response which if its an error, will probably have error in it, so we don't need to further increase our error hits anymore...

🧐